it/router/AccountTypeForCredIdFeature.scala

```diff
@@ -81,10 +81,10 @@ class AccountTypeForCredIdFeature extends StubbedFeatureSpec with CommonStubs {
       And("user's enrolments should be fetched from Auth")
       verify(getRequestedFor(urlEqualTo("/auth/enrolments-uri")))
 
-      And("user's details should not be fetched from User Details")
-      verify(1, getRequestedFor(urlEqualTo("/user-details-uri")))
+      And("user's details should be fetched from User Details")
+      verify(getRequestedFor(urlEqualTo("/user-details-uri")))
 
-      And("sa returns should be fetched from Sa micro service")
+      And("sa returns should be fetched from SA micro service")
```

Comment:
while you're changing `Sa` -> `SA`, you could probably also change `sa` -> `SA` (as in `sa returns`)